docs/usage-instructions-v2.md

```diff
@@ -135,6 +135,29 @@ for group in groups:
     print(group)
 ```
 
+## Disabling SSL Verification
+
+In environments where SSL inspection is enforced at the firewall, the UMAPI client can encounter the following error:
+
+2017-07-07 09:01:37 4916 CRITICAL main - UMAPI connection to org id 'someUUIDvalue@AdobeOrg' failed: [SSL: CERTIFICATE_VERIFY_FAILED] 
+
+This is because the requests module is not aware of the middle-man certificate required for SSL inspection.  The recommended solution to this problem is to specify a path to the certificate bundle using the  REQUESTS_CA_BUNDLE environment variable (see https://helpx.adobe.com/enterprise/kb/UMAPI-UST.html for details).  However, in some cases following these steps does not solve the problem.  The next logical step is to disable SSL inspection on the firewall for the UMAPI traffic.  If, however, this is not permitted, you may work around the issue by disabling SSL verification for user-sync.  
+
+Disabling the verification is unsafe, and leaves the umapi client vulnerable to middle man attacks, so it is recommended to  avoid disabling it if at all possible.  The umapi client only ever targets two URLs - the usermanagement endpoint and the ims endpoint - both of which are secure Adobe URL's.  In addition, since this option is only recommended for use in a secure network environment, any potential risk is further mitigated.
+
+To bypass the ssl verification, update the umapi config as follows:
+
+```yaml
+server:
+  ssl_verify: False
+```
```

Comment:
We shouldn't reference the User Sync Tool config here. When we make the update to the UST we should document the configuration changes there. What we should document here is how to disable SSL verification in the client itself i.e. create a `Connection` object with `ssl_verify=False`

Reply:
Whoops - I was looking at the YAML doc at the top and didn't realize it was used differently from UST in this example.  I replaced the YAML reference with a code example.